### ISSUE TITLE
Tips for training tabnet on a small scale dataset? 

### ISSUE THREAD

**Issue by puzzlecollector** [dreamquark-ai/tabnet#397]
@Optimox Do you have any tips when training TabNet on a small scale dataset for classification. By small scale I mean a train dataset with 3000-4000 datapoints.

**Comment by Optimox** [comment]
Not really, I think with a small dataset you should aim for a small architecture n_step=1 or 2 max.

You might also want to lower the batch size a bit so that it's still <10% of the original dataset.

Deep Learning models might not be the best choice with a very small dataset, but you can always give tabnet a try.

If you can comment with the scores you get with a simple benchmark with XGBoost and a Logistic Regression that would be great.

**Comment by puzzlecollector** [comment]
Thank you I will give it a try and will share the results if possible :)
